WordPress/src/main/res/layout/reader_recommended_blos_card.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/interest_container"
```

Comment:
`interest_container` doesn't seem to represent this container correctly. Seems like we can remove it. Wdyt?

Reply:
we can remove it